backend/python/app/resources/create_user_dto.py

```diff
@@ -5,3 +5,16 @@ def __init__(self, first_name, last_name, email, role, password):
         self.email = email
         self.role = role
         self.password = password
+
+    def validate():
+        if self.first_name == null || type(self.first_name) != str:
```

Comment:
```suggestion
        if self.first_name == None || type(self.first_name) != str:
```
Python equivalent is `None` :)